service/topology_coordinator.cc

```diff
@@ -152,7 +152,7 @@ static future<locator::tablet_replica_set> upsize_tablet_replicas_to_rf(const lo
 
         // Select subset of nodes to balance.
 
-    load_balancer_stats_manager stats;
+    load_balancer_stats_manager stats("upsize_tablet_replicas_to_rf");
```

Comment:
Why do we need separate stats manager for rf changes?

Reply:
We'll split this PR into 2:
1. here we'll only leave the request processing path and stub `reallocate_tablets_for_new_rf`
2. in https://github.com/scylladb/scylladb/pull/17116 @KrzaQ will implement `reallocate_tablets_for_new_rf`, 

I'll remove the 2nd part from this PR and @KrzaQ will address this comment in https://github.com/scylladb/scylladb/pull/17116